src/components/CurrentWalletBalance.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+// eslint-disable-next-line no-restricted-imports
```

Comment:
when we add lint exceptions inline we should metion why. Something like " Ignore the error here b/c we explicitly want native Text components for XYZ"

Reply:
Ah ok. My plan was to use https://github.com/Expensify/App/issues/6585 to go through each case, figure out if it could be refactored to use our components, and if not leave a comment on why. Does that work?

Reply:
Hmm would the tests for this PR fail if you removed it? If not I think the best practice would be to remove it then add them when you do the case by case along with the explanation. 

If it would break the tests then I guess it's fine, just remember to update all of them. I can't tell you how many place there are comments like "update this when this issue is done" in the code that never got updated 😅

Reply:
It does break the tests unfortunately. I can add `TODO` comments for each of them if you think that's better? 

I just figured it was better to have this PR focused on just updating the ESLint config version and suppressing the current errors that would result. Then have a separate PR for addressing our current uses of these components.

Reply:
Ah ok, I guess it's fine as is. 👍🏽

Reply:
Great, thanks! I'll make sure to address them all in that follow up issue.